### PR TITLE
chore: `make devcluster` to rebuild bindings before harness and webui.

### PR DESCRIPTION
**Pull request by ioga**
## Description

Existing setup leaves users exposed to stale bindings build. Let's have the harness and webui build keybindings rebuild the proto bindings as well.

## Commentary (optional)
https://hpe-aiatscale.slack.com/archives/C04TUNDGKMF/p1706040175088669

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: tools/devcluster.yaml [hunk 1]
```diff
@@ -2,9 +2,9 @@
 startup_input: "p"
 
 commands:
-  p: make -C harness clean build  # rebuild Python
-  w: make -C webui build          # rebuild Webui
-  c: make -C docs build           # rebuild doCs
+  p: make -C bindings build && make -C harness clean build  # rebuild Python
+  w: make -C bindings build && make -C webui build          # rebuild Webui
```
Comment (rb-determined-ai) [inline tools/devcluster.yaml:6]: make -C bindings build/python && make -C harness clean build
make -C bindings build/typescript && make -C webui build

File: tools/devcluster.yaml [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
+  c: make -C docs build  # rebuild docs
```
Comment (rb-determined-ai) [inline tools/devcluster.yaml:7]: the capital letter was the mnemonic, same with Python and Webui

Comment (ioga) [reply tools/devcluster.yaml:7]: ha

File: tools/devcluster.yaml (hunk 1 above)
Comment (rb-determined-ai) [reply tools/devcluster.yaml:6]: I don't feel strongly about this

**Review by rb-determined-ai — APPROVED** [review]

File: tools/devcluster.yaml (hunk 2 above)
Comment (rb-determined-ai) [reply tools/devcluster.yaml]: it works better in Warcraft III where they use yellow letters in a white font.  But with ascii, you have to work with what you got

**Pull request merged**